src/dealDashboard/dealDashboard.scss

```diff
@@ -1,6 +1,51 @@
 @import "src/styles/styles.scss";
 @import "src/resources/elements/primeDesignSystem/styles";
-
+.ppopup-dialog.executeSwap {
```

Comment:
this should be in its own modal dialog component

Reply:
Done